rust/otap-dataflow/crates/config/src/pipeline.rs

```diff
@@ -612,11 +622,207 @@ impl Default for PipelineConfigBuilder {
     }
 }
 
+/// Service-level telemetry configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct ServiceConfig {
+    /// The telemetry backend to which to report metrics.
+    #[serde(default)]
+    pub telemetry: TelemetryConfig,
+}
+
+impl Default for ServiceConfig {
+    fn default() -> Self {
+        Self {
+            telemetry: TelemetryConfig::default(),
+        }
+    }
+}
+
+/// Telemetry backend configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct TelemetryConfig {
+    /// The size of the reporting channel.
+    #[serde(default = "default_reporting_channel_size")]
+    pub reporting_channel_size: usize,
+    /// The interval at which metrics are flushed and aggregated by the collector.
+    #[serde(default = "default_flush_interval")]
+    pub flush_interval: std::time::Duration,
```

Comment:
I understand that these two parameters relate to the internal mechanism of the multivariate SDK. It will help to rename this reporting_interval to align it with reporting_channel_size. I 
```suggestion
    /// The size of the reporting channel, measured in the number of internal metric
    /// events shared across all cores.
    #[serde(default = "default_reporting_channel_size")]
    pub reporting_channel_size: usize,
    /// The interval at which metrics are flushed and aggregated by the collector.
    #[serde(default = "default_reporting_interval")]
    pub reporting_interval: std::time::Duration,
```